custodian/policies/guardduty/test-event/cryptotest.json

```diff
@@ -0,0 +1,172 @@
+{
```

Comment:
Is it possible to add a comment in here just to make clear this is a test event?

Reply:
Thanks, JSON doesn't support comments, however I've renamed the file name to make it clearer.